cloudsim-plus-testbeds/nb-configuration.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is a Netbeans specific file.
Include the following line inside the .gitignore to ignore it from the commit:
**/nb-configuration.xml

After that, remove the file with (the  --cached  will not remove the file from the disk):
git rm --cached cloudsim-plus-testbeds/nb-configuration.xml